lunar/distribution.yaml

```diff
@@ -710,6 +710,22 @@ repositories:
       url: https://github.com/orocos/orocos_kinematics_dynamics.git
       version: master
     status: maintained
+  pcl_msgs:
+    doc:
+      type: git
+      url: https://github.com/ros-perception/pcl_msgs.git
+      version: indigo-devel
+    release:
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/ros-gbp/pcl_msgs-release.git
+      version: 0.2.0-0
+    source:
+      test_pull_requests: true
```

Comment:
I am not sure you have followed [these instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide the necessary access for the ROS buildfarm GitHub user to the org unit yet?